api/report/engine/b_rep_fns.c

```diff
@@ -1412,67 +1554,97 @@ U_ch *RPF_SimNorm(U_ch** args)
  */
 U_ch *RPF_SimNIter(U_ch** args)
 {
-    int		t;
+    int		niter;
     char	buf[128];
 
+    niter = RPF_SimNIterInt(args);
+    if(niter < 0) return(SCR_malloc(1));
+
+    // Return niter in t
+    sprintf(buf, "%d", niter);
+    return(SCR_stracpy(buf));
+}
+
+
+/**
+ *  Subfunction of RPF_SimCpu().
+ *  
+ *  @param [in] args    char**  period  (eg: args[0] => "2001Y1")
```

Comment:
Sorry but I don't understand why `args` is a `char**` and not a simple `char*` if it represents a period.
If `args` represents several input, what are they?

Reply:
I fully understand your question. In fact, all report functions must have the same signature because they are **called by the same instruction:** see in `b_rep_engine.c` the function `RP_fneval().` You can see at line 979 the call:
```
*res = (*RP_FNS[i].fn)(tbl + 1);
```
where `RP_FNS `is the table of the @functions defined in `b_rep_syntax.c`. Each function MUST therefore have the same signature, hence the `char**`. But in the current function, we only use `args[1]`.

Reply:
> But in the current function, we only use args[1].

args[0] you mean ?